<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_exp4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,15">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;网络嗅探器&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_exp4_controller" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,3,1,2,1,3,1">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;监听地址&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_exp4_localIp"/>
             </item>
             <item>
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>次数</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_exp4_Count">
               <property name="inputMask">
                <string/>
               </property>
               <property name="placeholderText">
                <string>负数表示不限</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;次&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_exp4_start">
               <property name="text">
                <string>开始</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_exp4_saveToFile">
               <property name="text">
                <string>保存到文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;本功能需要管理员权限,Windows上没办法用Raw Socket抓 MAC 层的数据包，只能抓到IP层及以上的数据包！！！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_exp4_result">
            <column>
             <property name="text">
              <string>类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>From</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>To</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>内容</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>备注</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
